### PR TITLE
PayPal button added

### PR DESCRIPTION
**Pull request by mrSidSat**
Issue: #1501 

paypal button is added .

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi @mrSidSat thank you for your work. I have a few comments:

1. there are some unnecessary changes in sbuttons.css. Please make sure you're not manually making any edits.
2. There are a lot of changes in `buttons.json`, seems to be due to your editor's formatting. this can be a big issue to other contributors as this change will propagate.
3. There are changes in `website/package-lock.json

Here's what you need to do:

1. Fix the comments I added in `_paypal.less`.
2. run `npm run compile` and `npm run clean-css`.
3. revert `buttons.json` to the original format and just add your button.
4. remove all changes in `website/package.json`.
5. commit your changes.

**Comment by mrSidSat** [comment]
Thanks for the review , would do the changes ASAP

**Comment by mrSidSat** [comment]
All the required changes are made

**Review by shahednasser — CHANGES_REQUESTED** [review]
All looking great @mrSidSat but there's one problem I just noticed which is for rounded buttons the icon is now too close to the corner. So, set `right: 5px;` for rounded buttons only.

Don't forget to run `npm run compile` and `npm run clean-css` before committing the changes.

**Comment by mrSidSat** [comment]
The changes are made...

File: dist/sbuttons.css [hunk 1]
```diff
@@ -4262,8 +4262,8 @@ a.yellow-btn:active:not(.fill-color-btn) {
 }
 .icon-slide-btn.icon-down:hover .btn-label,
 .icon-slide-btn.icon-down:active .btn-label {
-  -webkit-transform: translate3d(0, 150%, 0);
-  transform: translate3d(0, 150%, 0);
+  -webkit-transform: translate3d(0, 200%, 0);
+  transform: translate3d(0, 200%, 0);
```
[the remark this answers is not in the document]
Comment (shahednasser) [reply dist/sbuttons.css]: I have noticed @RajMhatre20 as I have seen the same issue in other PRs, thank you for pointing it out as well!

**Pull request merged**